docs/docs/Configuration/configuration-custom-database.mdx

```diff
@@ -4,174 +4,154 @@ slug: /configuration-custom-database
 
-  The hostname in your connection string depends on how you're running PostgreSQL:
+    The hostname in your connection string depends on how you're running PostgreSQL:
 
-  - If you're running PostgreSQL directly on your machine, use `localhost`.
-  - If you're running PostgreSQL in Docker Compose, use the service name, such as `postgres`.
-  - If you're running PostgreSQL in a separate Docker container with `docker run`, use the container's IP address or network alias.
+    - If you're running PostgreSQL directly on your machine, use `localhost`.
+    - If you're running PostgreSQL in Docker Compose, use the service name, such as `postgres`.
+    - If you're running PostgreSQL in a separate Docker container with `docker run`, use the container's IP address or network alias.
+    - If you're running a cloud-hosted PostgreSQL, your provider will share your connection string, which includes a username and password.
 
-3. Create a Langflow `.env` file if you don't already have one:
+3. Create a Langflow `.env` file.
 
     ```
     touch .env
     ```
 
     You can use the [`.env.example`](https://github.com/langflow-ai/langflow/blob/main/.env.example) file in the Langflow repository as a template for your own `.env` file.
 
-4. In your `.env` file, set `LANGFLOW_DATABASE_URL` to your your PostgreSQL connection string:
+4. In your `.env` file, set `LANGFLOW_DATABASE_URL` to your PostgreSQL connection string.
 
     ```text
     LANGFLOW_DATABASE_URL="postgresql://user:password@localhost:5432/dbname"
     ```
 
-5. Save your changes, and then start Langflow with your `.env` file:
+5. Save your changes, and then start Langflow with your `.env` file.
 
     ```bash
     uv run langflow run --env-file .env
     ```
 
 6. In Langflow, run any flow to create traffic.
 
-7. Inspect your PostgreSQL deployment's tables and activity to verify that new tables and traffic were created after you ran a flow.
+7. Inspect your PostgreSQL database's tables and activity to verify that new tables and traffic were created after you ran a flow.
 
-## Example Langflow and PostgreSQL docker-compose.yml
+## Deploy Langflow and PostgreSQL containers with docker-compose.yml
 
-Docker Compose creates an isolated network for all services defined in `docker-compose.yml`. This ensures that the services can communicate with each other using their service names as hostnames, such as `postgres` in the database URL.
-In contrast, if you run PostgreSQL separately with `docker run`, it launches in a different network than the Langflow container, and this prevents Langflow from connecting to PostgreSQL using the service name.
+Use the [`docker-compose.yml`](https://github.com/langflow-ai/langflow/blob/main/docker_example/docker-compose.yml) file in the Langflow repository to launch Langflow and PostgreSQL containers in the same Docker network, ensuring proper connectivity between services.
```

Comment:
```suggestion
Launching Langflow and PostgreSQL containers in the same Docker network ensures proper connectivity between services.
For an example, see the [`docker-compose.yml`](https://github.com/langflow-ai/langflow/blob/main/docker_example/docker-compose.yml) file in the Langflow repository.
```

I only make this suggestion because they don't have to use that specific docker-compose; they can just use it as an example to set up their own docker-compose.